### PR TITLE
[1LP][RFR] execute cb on service vm

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <ndhandre@redhat.com>

Purpose or Intent
=================

{{pytest: cfme/tests/automate/custom_button/test_service_vm_custom_button.py::test_custom_button_automate_service_vm}}

### CODE REVIEW

File: cfme/tests/automate/custom_button/test_service_vm_custom_button.py [hunk 1]
```diff
@@ -228,3 +228,54 @@ def test_custom_button_with_dynamic_dialog_vm(
...
+        button = button_group.buttons.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            system="Request",
+            request="InspectMe",
+        )
+        request.addfinalizer(button.delete_if_exists)
+
+    # Check for UI and SSUI destinations.
+    for context in [ViaUI, ViaSSUI]:
```
Comment (john-dupuy) [inline cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: @digitronik thoughts on parameterizing this test case against `context`?

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: right, we have a tradition to parametrize a test with ViaUI` and `ViaSSUI` with the help of `santaku`. but I'm thinking to make it more simple and inclusive.

1. In this case, it's possible also but no need of this as we are checking a single button on different `destinations`.
2. If we did, test time will dobule; which is not good when the test needs more `setup steps` or `timing`. `service_vm` fixture is in `functional` scope. I won't want to spin VM just for button execution on different destinations.

Comment (john-dupuy) [reply cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: That makes sense, thanks for the explanation!

File: cfme/tests/automate/custom_button/test_service_vm_custom_button.py [hunk 2]
```diff
@@ -228,3 +228,54 @@ def test_custom_button_with_dynamic_dialog_vm(
             submit = "submit" if context is ViaUI else "submit_request"
             getattr(view, submit).click()
             log.wait_for_log_validation()
+
+
+@pytest.mark.meta(automates=[1427430, 1450473])
```
Comment (mshriver) [inline cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: 1454910?

Comment (mshriver) [reply cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: Talked to Nikhil, its a clone.

**Pull request merged**